Sources/PackageModel/PackageReference.swift

```diff
@@ -29,13 +27,8 @@ public struct PackageReference: Codable {
         case remote
     }
 
-    /// Compute identity of a package given its URL.
-    public static func computeIdentity(packageURL: String) -> String {
-        return computeDefaultName(fromURL: packageURL).lowercased()
-    }
-
     /// Compute the default name of a package given its URL.
-    public static func computeDefaultName(fromURL url: String) -> String {
+    internal static func computeDefaultName(fromURL url: String) -> String {
```

Comment:
This will break clients that show a package name before the package has been loaded.  What is the new API for them to use?

Reply:
could PackageReference expose the identity it is initialized with and such clients can work off of that?

Reply:
This is about being able to come up with a default name to show before the package has been loaded.  We could show the raw identity string, but it would be good to have a centralized place (of some form) to show the "display name" to use for a particular identity.

Reply:
I'd say we should keep this `public` in this case and figure this out separately as it is not strictly needed for Mattt to make progress.

Reply:
That would make it easier, but in the short term we can also switch clients to using `PackageIdentity.description` and add back a minimal API later.

Reply:
```suggestion
    public static func computeDefaultName(fromURL url: String) -> String {
```

I agree with @neonichu's take. How we decide to migrate this particular API is tertiary and severable from the rest of this PR. If we want to change the public API (and I think that we should), we can always do that later.